gocourse02/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import "fmt"
+
+// Struct that represents an Animal
+type Animal struct {
+	name      string // Name of the animal
+	species   string // Species of the animal
+	inTheCage bool   // Flag indicating whether the animal is in a cage
+}
+
+// Struct that represents a Zoo, containing multiple animals
+type Zoo struct {
+	Animals []Animal // Slice of animals in the zoo
+}
+
+// Struct that represents a Cage, which can hold an animal
+type Cage struct {
+	name   string // Name of the cage
+	animal string // Species of the animal currently in the cage
+}
+
+// Struct that represents all the cages in the zoo
+type AllCages struct {
+	Cages []Cage // Slice of cages
+}
+
+// Struct that represents a Zookeeper
+type Zookeeper struct {
```

Comment:
:memo: Якщо `zookeeper` - одне єдине слово, то ти написав правильно. Якщо два, то правильніше ось так:
```suggestion
type ZooKeeper struct {
```

Reply:
Одне https://www.merriam-webster.com/dictionary/zookeeper